cmd/infra/aws/create.go

```diff
@@ -50,16 +58,20 @@ func NewCreateCommand() *cobra.Command {
 
 	opts := CreateInfraOptions{
 		Region: "us-east-1",
+		Name:   "example",
 	}
 
 	cmd.Flags().StringVar(&opts.InfraID, "infra-id", opts.InfraID, "Cluster ID with which to tag AWS resources (required)")
 	cmd.Flags().StringVar(&opts.AWSCredentialsFile, "aws-creds", opts.AWSCredentialsFile, "Path to an AWS credentials file (required)")
 	cmd.Flags().StringVar(&opts.OutputFile, "output-file", opts.OutputFile, "Path to file that will contain output information from infra resources (optional)")
 	cmd.Flags().StringVar(&opts.Region, "region", opts.Region, "Region where cluster infra should be created")
 	cmd.Flags().StringSliceVar(&opts.AdditionalTags, "additional-tags", opts.AdditionalTags, "Additional tags to set on AWS resources")
+	cmd.Flags().StringVar(&opts.Name, "name", opts.Name, "A name for the cluster")
```

Comment:
may be clarify what a name for the cluster means? e.g "this is added as a subdomain qhen creating ingress resources"

Reply:
This is the same flag you would pass to `create cluster` and I copied the description from there hoping people would understand they are the same thing.  While we only use the `name` for DNS at `create infra` time, we use it for other things at `create cluster` time e.g. determining the namespace for the HCP.